Sources/Application/Routes/MarkdownRoutes.swift

```diff
@@ -22,19 +22,14 @@ func initializeMarkdownRoutes(app: App) {
 
     app.router.setDefault(templateEngine: KituraMarkdown())
 
-    app.router.get("/docs") { _, response, next in
-        response.headers["Content-Type"] = "text/html"
-        try response.render("/docs/index.md", context: [String:Any]())
-        response.status(.OK)
-        next()
-    }
-
     app.router.get("/docs/*") { request, response, next in
         response.headers["Content-Type"] = "text/html"
-        if request.urlURL.path != "/docs/" {
+        if request.urlURL.path == "/docs/" || request.urlURL.path == "/docs" {
+            try response.render("/docs/index.md", context: [String:Any]())
+            response.status(.OK)
+        }else {
             try response.render(request.urlURL.path, context: [String:Any]())
             response.status(.OK)
         }
-        next()
```

Comment:
We don't want to call next here since the static file server will match this route and overwrite our render. This is what was causing the original problem. Could you add `try response.end()` instead so the handler knows the route has completed?